src/main/kotlin/me/partlysanestudios/partlysaneskies/features/economy/auctionhousemenu/AuctionHouseGui.kt

```diff
@@ -166,8 +162,7 @@ class AuctionHouseGui(defaultAuctionInventory: IInventory) : WindowScreen(Elemen
 
 
     companion object {
-        @SubscribeEvent
-        fun onGuiOpen(event: ClientTickEvent) {
+        fun tick() {
```

Comment:
Can we please use the same name of the function everywhere and not tick, run, check, what ever? Ty!

Reply:
Smh he doesn't like originality

Reply:
I just don't like the inconsistency lol